.setup/add_sample_courses.py

```diff
@@ -6,12 +6,20 @@
 import json
 import os
 import pwd
+import sys
 
-submitty_repository = "/usr/local/submitty/GIT_CHECKOUT_Submitty"
-submitty_install_dir = "/usr/local/submitty"
-submitty_data_dir = "/var/local/submitty"
-sample_dir = submitty_install_dir + "/sample_files/sample_assignment_config"
-hwphp = (pwd.getpwnam("hwphp").pw_uid, grp.getgrnam("hwphp").gr_gid)
+SUBMITTY_REPOSITORY = "/usr/local/submitty/GIT_CHECKOUT_Submitty"
+SUBMITTY_INSTALL_DIR = "/usr/local/submitty"
+SUBMITTY_DATA_DIR = "/var/local/submitty"
```

Comment:
Except we'd have to then copy these file over to /usr/local/submitty/.setup first. We should then also be having CONFIGURE_SUBMITTY.sh write to a json file (or some other type of easily parsed file) that contains these things instead of doing more sed replaces (which is brittle and hard to change if need be).